spec/neo4j/driver/dev_manual_examples_spec.rb

```diff
@@ -23,7 +23,7 @@
     after { driver2.close }
 
     let(:driver2) { Neo4j::Driver::GraphDatabase.driver(uri, auth_tokens, config) }
-    let(:auth_tokens) { Neo4j::Driver::AuthTokens.basic('neo4j', 'pass') }
+    let(:auth_tokens) { basic_auth_token }
```

Comment:
maybe make here an exception as this is a documentation spec taken from Neo4j Driver Manual and there is merit in showing how this works without chasing custom helper methods.